app/views/components/super-navigation-header/_header-logo.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="govuk-header__logo govuk-header__super-navigation-header-logo">
+  <a href="<%= content_for?(:homepage_url) ? yield(:homepage_url) : "https://www.gov.uk/" %>"
+      title="<%= content_for?(:logo_link_title) ? yield(:logo_link_title) : "Go to the GOV.UK homepage" %>"
+      class="content govuk-header__link govuk-header__link--homepage"
+      id="logo"
```

Comment:
Once the smokey test has been updated, we can get rid of this `id="logo"` as it's not used for anything.